frontends/ol-ckeditor/src/webpack-utils.js

```diff
@@ -3,6 +3,13 @@ const {
   CKEditorTranslationsPlugin,
 } = require("@ckeditor/ckeditor5-dev-translations")
 
+/* TODO
+  Is this config used, ie. do we ever prebuild ol-ckeditor?
+  Currently this is built from source from the mit-open entrypoint.
```

Comment:
We do use this: https://github.com/mitodl/mit-open/blob/3b55e0e8057a7a5916a975133680a3a9f8d3330a/frontends/mit-open/webpack.config.js#L152

We do not pre-build CKEditor—this isn't a config, it's utils for including CKEditor into a webpack build.